src/config.rs

```diff
@@ -3011,6 +3011,29 @@ lazy_static! {
     pub static ref TIKVCONFIG_TYPED: ConfigChange = TiKvConfig::default().typed();
 }
 
+fn serde_to_online_config(mut name: String) -> String {
```

Comment:
How about moving the following code into this function
https://github.com/tikv/tikv/blob/927b9a7639d144053641ea6b3124074699695c9e/src/config.rs#L3022-L3026